src/charm.py

```diff
@@ -250,12 +243,6 @@ def _on_check_config(self, event: ActionEvent) -> None:
             {"result": output, "error-message": err, "valid": False if err else True}
         )
 
-    def _on_ingress_ready(self, event: IngressPerAppReadyEvent):
```

Comment:
Does this charm do anything with the ingress address? Apparently not.

Reply:
It's used by `@property def _external_url(self)`.